### ISSUE TITLE
how dare you!?

### ISSUE THREAD

**Issue by multiscan** [keybase/client#25243]
Keybase GUI Version: 6.0.1-20220609162338+0fc9feea3d

in the last 6 months I've been bothered every 15 minutes by a popup window of keybase asking me to upgrade. I heard of issues with osx 10. Therefore I kept clicking "Cancel" until by mistake I click OK and now keybase is no longer working!
I have tried to uninstall it and re-install it via brew and I read the following message:

```
Warning: You are using macOS 10.15.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from MacHomebrew on Twitter.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.
```

Therefore you knew that the upgraded version of the program would never work on my laptop. Nevertheless you bothered me for moths with a popup dialog asking me to upgrade!
This is pure stalking. You idiots!

**Comment by TheButterZone** [comment]
I just updated that same Keybase version, working fine so far in OSX 10.15.7, which OSX you got?

**Comment by multiscan** [comment]
I had the same version of  OSX (if I remember correctly). Now, due to keybase I had to upgrade to OSX 12.0.1. 
In fact the GUI app was working fine it was just the finder integration that stopped working. And this is essential for me as all my deployment script are making use of the /keybase mount point.

**Comment by TheoOliveira** [comment]
Yeah, because the open sourced team is so much interested in stalking you to upgrade your version.... wow, that karen....

**Comment by allanpaiste** [comment]
There are older releases available for you to downgrade when you encounter issues: 

https://prerelease.keybase.io/darwin-test/index.html

(I had to downgrade due to 6.0.3 not working with macOS 12).

As for the conduct here: the title of this issue doe draw attention for all the wrong reasons and impedes people to provide proper help (they'll end up here only due to curiousity like I just did). Taking emotion out from the post would have gotten you to get help much faster.
